bccsp/sw/fileks_test.go

```diff
@@ -30,7 +30,7 @@ func TestInvalidStoreKey(t *testing.T) {
 
 	ks, err := NewFileBasedKeyStore(nil, filepath.Join(tempDir, "bccspks"), false)
 	if err != nil {
-		fmt.Printf("Failed initiliazing KeyStore [%s]", err)
+		fmt.Printf("Failed initiliazing KeyStore [%s]\n", err)
```

Comment:
Why the new line? Why the odd formatting of the error? Why printf and exit instead of failing the test with an assertion or `Fatalf`?

Reply:
addressed in #936